opencog/query/PatternMatchEngine.cc

```diff
@@ -780,8 +781,8 @@ bool PatternMatchEngine::tree_compare(const PatternTermPtr& ptm,
 	if (not match) return false;
 
 	LAZY_LOG_FINE << "depth=" << depth;
-	logmsg("> tree_compare", hp);
-	logmsg(">           to", hg);
```

Comment:
again -- removing important formating!   These align the atoms so that they can be directly compared!